### ISSUE TITLE
Что делает AUTO_UPGRADE?

### ISSUE THREAD

**Issue by bd100b** [shamhi/HamsterKombatBot#101]
Хотелось бы узнать детальнее что делает AUTO_UPGRADE? Написано Улучшает пассивный заработок. Значит ли это что оно прокачивает мои карточки майнинга?

**Comment by Prohi1** [comment]
Да, он автоматически улучшает карточки исходя из наиболее выгодных позиций

**Comment by tancyit3** [comment]
> Да, он автоматически улучшает карточки исходя из наиболее выгодных позиций

Точно? Раньше просто по другому улучшал и в целом было целесообразнее без автоматического апгрейда работать

**Comment by Prohi1** [comment]
Его можно отключить использовать бота только для тапов

**Comment by shamhi** [comment]
> > Да, он автоматически улучшает карточки исходя из наиболее выгодных позиций
> 
> Точно? Раньше просто по другому улучшал и в целом было целесообразнее без автоматического апгрейда работать

раньше да, но сейчас он выбирает лучшие

**Comment by ButCheR-PRO** [comment]
Логику автоулучшения карт можем обсудить здесь: https://github.com/shamhi/HamsterKombatBot/issues/142
